src/main/java/org/jenkinsci/plugins/workflow/steps/FileExistsStep.java

```diff
@@ -78,7 +78,10 @@ public String getFile() {
         }
 
         @Override protected Boolean run() throws Exception {
-            if (StringUtils.isEmpty(file)) {
```

Comment:
What about `StringUtils.IsBlank(file)`?  That also return `true` for null.  Do we need to log that the file was `null` rather than `""`?

Reply:
I guess the distinction doesn't matter. I'll fix this up. The important part is to not throw an NPE due to the return statement

Reply:
Oh, I see now.  How about unifying these paths instead?
```
file = StringUtils.isEmpty(file) ? "" : file;
```

Reply:
No NPE: absolutely. Thanks!

Reply:
I just changed the approach and called it with a blank string in the event its blank or null